index.js

```diff
@@ -264,7 +265,7 @@ async function run() {
     // Get inputs
     const accessKeyId = core.getInput('aws-access-key-id', { required: false });
     const secretAccessKey = core.getInput('aws-secret-access-key', { required: false });
-    const region = core.getInput('aws-region', { required: true });
+    const region = core.getInput('aws-region', { required: false }) || AWS_CURRENT_REGION;
```

Comment:
Like I mentioned above, we should use `process.env.XXX` instead of a global variable (after all you're only using it on this line)

To add on to the discussion above regarding `AWS_REGION` vs `AWS_DEFAULT_REGION`... why not both? Action input > AWS_REGION > AWS_DEFAULT_REGION should work here no?